### PR TITLE
update name and type when showing catalogs

### PR DESCRIPTION
**Pull request by stephen-shelby**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [x] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|1|1|100.00%|

**Comment by stephen-shelby** [comment]
@Mergifyio backport branch-2.3

File: fe/fe-core/src/test/java/com/starrocks/analysis/ShowCatalogsStmtTest.java [hunk 1]
```diff
@@ -59,10 +59,10 @@ public void testShowCatalogsNormal() throws AnalysisException, DdlException {
         ShowExecutor executor = new ShowExecutor(ctx, stmt);
         ShowResultSet resultSet = executor.execute();
         ShowResultSetMetaData metaData = resultSet.getMetaData();
-        Assert.assertEquals(metaData.getColumn(0).getName(), "Catalog");
-        Assert.assertEquals(metaData.getColumn(1).getName(), "Type");
-        Assert.assertEquals(metaData.getColumn(2).getName(), "Comment");
-        Assert.assertEquals(resultSet.getResultRows().get(0).toString(), "[default, default, internal catalog]");
-        Assert.assertEquals(resultSet.getResultRows().get(1).toString(), "[hive_catalog_1, hive, hive_catalog]");
+        Assert.assertEquals("Catalog", metaData.getColumn(0).getName());
```
Comment (stephen-shelby) [inline fe/fe-core/src/test/java/com/starrocks/analysis/ShowCatalogsStmtTest.java:62]: @adzfolc Please remember how this method is used :)

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|1|1|100.00%|

File: fe/fe-core/src/test/java/com/starrocks/analysis/ShowCatalogsStmtTest.java [hunk 2]
```diff
@@ -59,10 +59,10 @@ public void testShowCatalogsNormal() throws AnalysisException, DdlException {
...
+        Assert.assertEquals("Type", metaData.getColumn(1).getName());
+        Assert.assertEquals("Comment", metaData.getColumn(2).getName());
+        Assert.assertEquals("[default_catalog, Internal, Internal Catalog]", resultSet.getResultRows().get(0).toString());
```
Comment (DorianZheng) [inline fe/fe-core/src/test/java/com/starrocks/analysis/ShowCatalogsStmtTest.java:65]: use `InternalCatalog.DEFAULT_INTERNAL_CATALOG_NAME`?

**Review by DorianZheng — APPROVED** [review]
LGTM

File: fe/fe-core/src/test/java/com/starrocks/analysis/ShowCatalogsStmtTest.java (hunk 1 above)
Comment (adzfolc) [reply fe/fe-core/src/test/java/com/starrocks/analysis/ShowCatalogsStmtTest.java:62]: > @adzfolc Please remember how this method is used :)

got

**Review by Astralidea — APPROVED** [review]

**Pull request merged**